packages/access-api/src/utils/context.js

```diff
@@ -44,6 +45,10 @@ export function getContext(request, env, ctx) {
       : UCANLog.debug()
 
   // Sentry
+  // eslint-disable-next-line no-console
+  console.warn('constructing toucan-js', {
```

Comment:
```suggestion
  console.log('constructing toucan-js', {
```

Reply:
I'll remove the console alltogether, as this was useful for debbugging something with `wrangler tail` but no longer